vendors/st/boards/stm32l475_discovery/aws_tests/application_code/main.c

```diff
@@ -109,6 +112,17 @@ int main( void )
 
     return 0;
 }
+
+/*-----------------------------------------------------------*/
+
+static void testRunnerHook( void * pvParameters )
```

Comment:
testRunnerTask is probably better?

Reply:
The name of the function will be updated. Thanks for the suggestion.